src/theme/palette.ts

```diff
@@ -0,0 +1,66 @@
+import { PaletteOptions } from "@mui/material"
+import { alpha } from "@mui/material"
+
+const saddleColors = {
+  indigo: "#4B11F2",
+  seafoam: "#06D7D7",
+  seaweed: "#00F4D7",
+  sunburn: "#E6AD76",
+  gold: "#FAEA5D",
+  white: "#FFF",
+  black: "#000",
+  grey70: "#252525",
+  grey50: "#404040",
+  grey25: "#7D7D7D",
+  grey5: "#A7A7A7",
+  grey1: "#E4E4E4",
+  indigo2: "#311188",
+  indigo3: "#121334",
+  indigo4: "#070713",
+  seafoam2: "#126969",
+  sunset: "#D96A3B",
+  sand1: "#E3D899",
+  sand2: "#FAF3CE",
+  sand3: "#FDFDF8",
+  gold2: "#817F48",
+  cherry: "#FB5A5A",
+  cherry2: "#68282F",
+}
+
+const GREY = {
```

Comment:
Changing this name to `GRAY_PALLETE` would help me understand it a bit more since it isn't exactly a single color.

Reply:
I recommend to use `GRAY_TONES` because there is `lightPalette and darkPalette name.